hw2/05-form.js

```diff
@@ -1 +1,27 @@
 // Add your code here
+const inputForm = document.getElementById("inputForm");
+
+const printForm = (event) => {
+  event.preventDefault();
+  const fullname = inputForm.elements[0].value;
+  const email = inputForm.elements[1].value;
+  const registrationStatus = inputForm.elements[2].value;
```

Comment:
instead of using elements[x].value, to get your values, use one of the DOM methods.  That way if you ever need to add a extra label to the form, you just add one line, instead of having to search for the element[x] values that will need to be updated due to your change.

Reply:
Haha. I commented exactly the opposite on your PR. I guess different approaches for the same solution.